docs/UsersGuide/user/enum-arr-ser.md

```diff
@@ -94,8 +94,8 @@ These can all be designed and used in the modeling of F´ and the C++ implementa
 
 ### Enums
 
-Enums are a fix set of values that can be referred to by name, but are store and transmitted as an integer type. The
-auto-generator produces a class to wrap the type providing convenience functions.
+Enums are a fixed set of values that can be referred to by name, but are stored and transmitted as an integer type. The
+auto-generator produces a class to wrap the type providing convenient functions.
```

Comment:
This change is a regression -- "convenience functions" is a term of art. See https://en.wikipedia.org/wiki/Convenience_function.

Reply:
Yes, I should fix this.